File: LektaAndroid/app/src/main/java/com/lekta/app/models/PaymentMethod.kt
package com.lekta.app.models

import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.filled.Money
import androidx.compose.material.icons.filled.QrCode
import androidx.compose.material.icons.filled.Waves
import androidx.compose.ui.graphics.vector.ImageVector

enum class PaymentMethod(val label: String, val icon: ImageVector) {
    QR_MP("QR MP", Icons.Default.QrCode),
    POINT_MP("Point MP", Icons.Default.Waves),
    CASH("Efectivo", Icons.Default.Money);
}

File: Models/PaymentMethod.swift
import Foundation

enum PaymentMethod: String, CaseIterable, Codable {
    case qrMP    = "QR MP"
    case pointMP = "Point MP"
    case cash    = "Efectivo"

    var icon: String {
        switch self {
        case .qrMP:    return "qrcode"
        case .pointMP: return "wave.3.right"
        case .cash:    return "banknote"
        }
    }
}
